pkgs/development/libraries/science/chemistry/tblite/python.nix

```diff
@@ -0,0 +1,33 @@
+{ buildPythonPackage
+, meson
+, ninja
+, pkg-config
+, tblite
+, cffi
+}:
+
+buildPythonPackage rec {
+  pname = tblite.pname;
+  version = tblite.version;
+
+  src = tblite.src;
+
+  nativeBuildInputs = [ meson ninja pkg-config ];
+
+  buildInputs = [ tblite ];
```

Comment:
You use the sources from `tblite` and the package as build inputs at the same time? Does that cause ofborg to end up with an infinite recursion error?

Reply:
I'm afraid I do not understand completely what you mean by this :grimacing:

Reply:
Do we need `buildInputs = [ tblite ];` here? We already use it as a source (` src=tblite.src`).

Reply:
Ah yes, we do. Building the python interface is a three step procedure:

  1. Build the Fortran lib (the original tblite derivation)
  2. Build C-Interface (`configurePhase` in `python.nix`)
  3. Build the Python package around the C-Interface

It explicitly requires to have completed step 1 first and install tblite with tblite.pc files. Thus, yes we need it.

Reply:
Now I see the recursion error of OfBorg. Didn't noticed at all, first. The issue is much simpler than this. In `python-packages.nix` i have to pass `tblite = pkgs.tblite` and `meson = pkgs.meson` instead of `super.meson`  and `super.tblite`. `super` is the python package set. Is fixed now